package.json

```diff
@@ -116,7 +116,7 @@
         "@rollup/plugin-json": "^6.0.1",
         "@rollup/plugin-node-resolve": "^15.2.3",
         "@sindresorhus/slugify": "^2.1.1",
-        "@spectrum-web-components/eslint-plugin": "file:./linters/eslint",
```

Comment:
Windows really did not like this file syntax. Using the local version number, we are still leveraging the internal package.

Reply:
Ye... There's a lot Windows does not like 😞 Thanks for this fix.